Benqi-Agents/DelegationMonitor/README.md

```diff
@@ -0,0 +1,33 @@
+
+# Delegations Monitor Agent
+
+## Description
+
+This agent detects users with a huge balance delegating their votes.
+
+## Supported Chains
+
+- Avalanche
+
+## Alerts
+
+
+- BENQI-2
+  - Fired when a user with a balance greater than the threshold delagates his votes.
+
+  - Severity is always set to "Info" 
+  - Type is always set to "Info" 
+  - Metadata includes:
+    * `delegator`: address of the user delegating his votes
+
+    * `fromDelegate`: address of the current delegate
+    * `toDelegate`: address of the new delegatee
+
+    * `balance`: balance of the delegator
+
+## Test Data
+
+The agent behaviour can be verified with the following transactions:
+- 0x712b53c81428662ed906b1571831f2865cd79a7c2d9514551199f45c0469b3fb(1 findings)
+- 0x194b328faf28f652179af93ed9c51944289d64750a93847977a5e4f9d86a4bf1(1 findings)
+- 0xc8295477c18cb35e3a4465c07b162c454683b68000ccfb84b7943ef698558d3b(0 findings)

```

Comment:
Please specify how to run this tests. They didn't work on Avalanche mainnet. Also I got 0 findings in each of them using `util.QI_CONTRACT` and Avalanche testnet.

PD: change the name of the address in utils to `TEST_QI_CONTRACT`